include/klee/Expr.h

```diff
@@ -176,7 +176,26 @@ class Expr {
 
 protected:  
   unsigned hashValue;
-  
+
+  /// Compares `b` to `this` Expr for structural equivalence ignoring
```

Comment:
@andreamattavelli Parent/children aren't really being excluded because `ReadExpr` traverses it's update list and that is a structural comparison (i.e. there are many update lists that are not structurally equivalent but are semantically equivalent).